src/inmanta/server/services/resourceservice.py

```diff
@@ -866,3 +870,59 @@ async def resource_did_dependency_change(
             for dependency, actions in (await self.get_resource_events(env, resource_id)).items()
             for action in actions
         )
+
+    @protocol.handle(methods_v2.resource_list, env="tid")
+    async def resource_list(
+        self,
+        env: data.Environment,
+        limit: Optional[int] = None,
+        first_id: Optional[str] = None,
+        last_id: Optional[str] = None,
+        start: Optional[str] = None,
+        end: Optional[str] = None,
+        filter: Optional[Dict[str, List[str]]] = None,
+        sort: str = "resource_type.desc",
+    ) -> ReturnValue[List[ResourceListElement]]:
```

Comment:
According to the docstring of this API endpoint `start` and `end` cannot be set at the same time, but these checks are missing here.

Reply:
It seems that change was not implemented. Did you overlook this?

Reply:
Ah sorry, I wanted to reply, but forgot it. There is a check for this in the data layer (`validate_paging_parameters`), and I've added test cases for it.
Do you think it should be checked in the `resourceservice` (as well)?

Reply:
ok